driver/src/main/scala/api/commands/rwcommands.scala

```diff
@@ -21,23 +21,23 @@ object GetLastError {
   sealed trait W
   case object Majority extends W
   case class TagSet(tag: String) extends W
-  case class WaitForAknowledgments(i: Int) extends W
+  case class WaitForAcknowledgments(i: Int) extends W
```

Comment:
I added the annotations for class and method. Thank you.
And I gave up compatibility between WaitForAcknowledgments and WaitForAknowledgments, as case-to-case inheritance is prohibited.